docker-hub/image-management.md

```diff
@@ -24,11 +24,15 @@ With the new Advanced Image Management dashboard, all of this is now simpler and
 
 To access the  Advanced Image Management dashboard:
 
-1. Log into [Docker Hub](https://hub.docker.com){: target="_blank" rel="noopener" class="_"}.
-2. Click **Repositories** from the main menu and select a repository from the list. This displays a banner indicating the number of inactive images in the repository.
-3. Click **View inactive images**. This displays the Advanced Image Management dashboard. It lists images in your repository and allows you to sort and filter images by tags, activity status, and the last activity date.
+1. Log in to [Docker Hub](https://hub.docker.com){: target="_blank" rel="noopener" class="_"}.
+2. Click **Repositories** from the main menu and select a repository from the list.
+3. Click **General** or **Tag**, and click **Go to Advanced Image Management** from the main menu, and select a repository from the list.
```

Comment:
**suggestion:** the tab says **Tags** (in plural), here it's stated in the singular form